LogicMonitor.DataSDK/Api/Logs.cs

```diff
@@ -37,9 +38,9 @@ public Logs(bool batch =true, int interval = 10, IResponseInterface responseCall
         /// </summary>
         /// <param name="message">Log Message.</param>
         /// <param name="resource">Resource object.</param>
-        public RestResponse SendLogs(string message, Resource resource, Dictionary<string, string> metadata = default,string timestamp=default)
+        public RestResponse SendLogs(string message, Resource resource, Dictionary<string, string> metadata = default,string timestamp=default, LogLevel logLevel=default)
```

Comment:
Here using "default" ignores user provided Loglevel Trace and replaces it with null.
We can use nullable like "LogLevel? logLevel". Please test it for null.